trust-center-agent/src/test/java/care/smith/fts/tca/consent/FhirConsentProviderTest.java

```diff
@@ -176,7 +186,12 @@ void noConsents(MockServerClient mockServer) {
     fhirConsentProvider =
         new FhirConsentProvider(
             httpClientBuilder.baseUrl(address).build(), policyHandler, pageSize);
-    Bundle bundle = gicsConsentGenerator.generateBundle(totalEntries, pageSize);
+    Bundle bundle =
```

Comment:
Recurring code, see  l152. Since it is just in tests, we might consider it okay, though.